lib/commands/hasFocus.js

```diff
@@ -27,19 +27,21 @@
  */
 
 let hasFocus = function (selector) {
-    let result = this.execute(function (selector) {
-        var focused = document.activeElement
+    return this.elements(selector).then((res) => {
```

Comment:
We looked at the code with @mattc41190 and I would leave it as is without a new `if` branch. Does anybody have any better suggestions?